decidim-meetings/app/forms/decidim/meetings/meeting_form.rb

```diff
@@ -19,8 +19,13 @@ class MeetingForm < Decidim::Form
       attribute :user_group_id, Integer
       attribute :online_meeting_url, String
       attribute :type_of_meeting, String
+      attribute :registration_type, String
+      attribute :registration_url, String
+      attribute :available_slots, Integer, default: 0
+      attribute :registration_terms, String
 
       TYPE_OF_MEETING = %w(in_person online).freeze
+      REGISTRATION_TYPE = %w(registration_disabled on_this_platform on_different_platform).freeze
```

Comment:
why re define the same constant in the model here? Can't we remove this constant?